README.md

```diff
@@ -32,12 +34,15 @@ for i in range(len(gdf_page)):
 | :-------------------------------------------- | ---------- | ---------------------------- |
 | PageRank                                      | Multi-GPU  |                              |
 | Personal PageRank                             | Single-GPU |                              |
+| Katz Centrality                               | Single-GPU |                              |
 | Jaccard Similarity                            | Single-GPU |                              |
 | Weighted Jaccard                              | Single-GPU |                              |
 | Overlap Similarity                            | Single-GPU |                              |
 | SSSP                                          | Single-GPU | Updated to provide path info |
 | BFS                                           | Single-GPU |                              |

```

Comment:
Should we mention that we also have a BSP variant for BFS in the Notes?

Reply:
done